packages/x-charts/src/context/AnimationProvider/AnimationProvider.tsx

```diff
@@ -1,59 +0,0 @@
-'use client';
-import * as React from 'react';
-import useEnhancedEffect from '@mui/utils/useEnhancedEffect';
-import { AnimationProviderProps } from './Animation.types';
-import { AnimationContext } from './AnimationContext';
-
-function AnimationProvider(props: AnimationProviderProps) {
-  // Part of the behavior was taken from:
-  // https://github.com/pmndrs/react-spring/blob/fd65b605b85c3a24143c4ce9dd322fdfca9c66be/packages/shared/src/hooks/useReducedMotion.ts
-  const { children, skipAnimation: inSkipAnimation } = props;
-
-  // Skip animation test/jsdom
-  const isAnimationDisabledEnvironment = typeof window === 'undefined' || !window?.matchMedia;
-
-  // We use the value of `isAnimationDisabledEnvironment` as the initial value of `skipAnimation` to avoid
-  // re-rendering the component on environments where matchMedia is not supported, hence skipAnimation will always be true.
-  const [skipAnimation, setSkipAnimation] = React.useState<true | undefined>(
-    process.env.NODE_ENV === 'test' ? isAnimationDisabledEnvironment || undefined : undefined,
-  );
-
-  useEnhancedEffect(() => {
-    if (isAnimationDisabledEnvironment) {
-      return undefined;
-    }
-
-    const handleMediaChange = (event: { matches: boolean | undefined }) => {
-      // This hook can remove animation but never activate it.
-      const inputValue = event.matches || undefined;
-      setSkipAnimation(inputValue);
-    };
-
-    const mql = window.matchMedia('(prefers-reduced-motion)');
-
-    handleMediaChange(mql);
-
-    // MatchMedia is not fully supported in all environments, so we need to check if it exists before calling addEventListener
```

Comment:
This is no longer true. All our supported platforms support `addEventListener` on `MediaQueryList`.

Reply:
Do you know where in CI it used to fail? CI seems to have passed without it.